pkgs/development/tools/haskell/tinc/default.nix

```diff
@@ -7,12 +7,12 @@
 }:
 mkDerivation {
   pname = "tinc";
-  version = "20160511";
+  version = "20160924";
   src = fetchFromGitHub {
-    owner = "sol";
+    owner = "robbinch";
```

Comment:
the last fix, you made seems reasonably. Is there a reasons, you did not made a pull request against the original project and propose your fork now?